services/workflows-service/src/customer/api-key/utils.ts

```diff
@@ -24,7 +26,7 @@ export const hashKey = async (key: string, salt?: string) => {
       return reject(new Error('Invalid key length'));
     }
 
-    bcrypt.hash(key, salt ?? SALT, (err, hashedKey) => {
+    bcrypt.hash(key, salt ?? SALT!, (err, hashedKey) => {
```

Comment:
Does TypeScript warn against `null` or `undefined` due to decoding being able to fail?

Reply:
validating the salt as well on startup when hashing random string
<img width="852" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/cfacb4c9-adee-4f59-b195-41e29b546623">

Reply:
yeah the salt has to be present cant get undefined